### PR TITLE
Update detail

### PR DESCRIPTION
**Pull request by gaurabpokharel**
contact-us

### CODE REVIEW

**Comment by ajaystha42** [comment]
@gaurabpokharel  please solve conflicts before sending PR.

**Comment by gaurabpokharel** [comment]
contact us page

File: html/contactUs.html [hunk 1]
```diff
@@ -92,7 +206,6 @@ <h4><a href="index.html">SoundWave LOGO</a></h4>
       $("#login").hide();
       $("#signup").hide();
       $("#userDetails").show();
-      $("#logout").show();
```
Comment (ajaystha42) [inline html/contactUs.html:95]: why removing this? @gaurabpokharel

File: php/login.php [hunk 2]
```diff
@@ -19,16 +19,22 @@
     // Getting Value from Form
     $email = mysqli_real_escape_string($con, $_POST["email"]);
     $password = mysqli_real_escape_string($con, $_POST["password"]);
-    
+
+
+    echo $email ;
```
Comment (ajaystha42) [inline php/login.php:24]: these are not necessary @gaurabpokharel

**Review by ajaystha42 — CHANGES_REQUESTED** [review]

**Pull request merged**